src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -793,34 +794,33 @@ export class AppComponent implements OnInit, AfterViewChecked {
     /**
      *
      */
-    keyEvent(e: KeyboardEvent): void {
-        const self = this;
-        let eString = this.shortcuts.buildEventString(e);
-        this.shortcuts.getEvent(eString).then((result) => {
-            if (result) {
-                let eventName = <string> result;
-                self.shortcutfunc[eventName]();
-                e.stopImmediatePropagation();
-            }
-        });
-    }
-
-    /**
-     * Helper to deselect messages when Ctrl + A is pressed
-     * inside the grid
-    */
-    keyUpEvent(e: KeyboardEvent): void {
-        let eString = this.shortcuts.buildEventString(e);
-        this.shortcuts.getEvent(eString).then((result) => {
+    async keyEvent(e: any): Promise<void> {
+        if (e.target.classList.contains('slick-cell')) {
+            $('#messages').css('user-select', 'none');
```

Comment:
What about at the very least setting the user-select back to text once you're done with the selection instead of doing that on every other event? That seems a like a lot of extra processing that most of the time won't actually be necessary.

Reply:
The selected text get unselected in that case. This is doing less work in fact, since it only switches the user selection once instead of twice